### ISSUE TITLE
Is there a similar control to TimePickerCtrl without the bounds of time?

### ISSUE THREAD

**Issue by swprojects** [wxWidgets/Phoenix#120]

**Comment by RobinD42** [comment]
Can you describe what you mean by "without the bounds of time"?

**Comment by swprojects** [comment]
Sorry for unclear question, I closed the question because I decided to create a wx.Control myself

I basically want a similar control to TimePickerCtrl but the values being just integers, rather than being DateTime values.

For example:  -55:20:90 + SpinButton

However, I want a single control rather than, say for example, TextCtrls bound to a SpinButton on a panel.

for example in the image below, I have a ListCtrl which brings up specific controls when editing the value rather than just editing the text.

![Image](https://cloud.githubusercontent.com/assets/10121727/17154533/57d5f5fa-5379-11e6-80b3-d4010e68ea1c.png)
